libs/gi/ui/src/components/artifact/editor/SubstatInput.tsx

```diff
@@ -129,26 +128,31 @@ export function SubstatInput({
                 </MenuItem>
               ))}
           </DropdownButton>
-          <CustomNumberInputButtonGroupWrapper
-            sx={{ flexBasis: 30, flexGrow: 1 }}
+          <Box
```

Comment:
Can you try putting the `NumberInputLazy` here without the `Box` wrapper? You can apply `sx` directly to the `NumberInputLazy`